src/components/PostPageDetail/index.js

```diff
@@ -105,12 +104,13 @@ const PostPageDetailIdComponent = (props) => {
   React.useEffect(() => {
     getComment();
   }, []);
+
   const handleVote = (data = {}) => {
     const upvote = data.upvotes ? data.upvotes : 0;
     const downvotes = data.downvotes ? data.downvotes : 0;
     setTotalVote(upvote - downvotes);
   };
-  const initial = async () => {
+  const initial = () => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `async` keyword from the `initial` function seems fine as there doesn't seem to be any asynchronous operations within the function. However, if there are any asynchronous operations that were removed in this PR, please ensure they are handled properly.



<!-- This is an auto-generated comment by OSS CodeRabbit -->